### ISSUE TITLE
Reconnect option

### ISSUE THREAD

**Issue by maciekfreak** [pdeschen/bancroft#13]
Is there any option to reconnect if gpsd isn't running?

From my tests I see that if gpsd isn't running before brancroft  there is no option to connect again..

**Comment by pdeschen** [comment]
@maciekfreak yeah it is one of the feature on my [todo list](https://github.com/pdeschen/bancroft#todos).  
How would you like to see it? An option `autoreconnect` where bancroft would perform liveness test and reconnect as necessary? Would only during start-up phase enough?

**Comment by maciekfreak** [comment]
Of course autoreconnect on every disconnect will be the best solution I think.

More data form SKY and TPV (like mode, lat err, lon err etc.)  will be also good feature. I can try to make pull request with this

**Comment by pdeschen** [comment]
@maciekfreak go for it!
